File: Lecture 5/lecture5_src/phonebook0.c
// Сохранение имён и номеров в CSV файл

#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    // Открытие CSV файла
    FILE *file = fopen("phonebook.csv", "a");

    // Ввод пользователем имени и номера
    char *name = get_string("Name: ");
    char *number = get_string("Number: ");

    // Вывод значений в файл
    fprintf(file, "%s,%s\n", name, number);

    // Закрытие CSV файла
    fclose(file);
}

File: Lecture 7/lecture7_src/4/phonebook0.py
# Запись имени и номера в CSV файл

import csv

# Открытие CSV файла
file = open("phonebook.csv", "a")

# Получение имени и номера
name = input("Name: ")
number = input("Number: ")

# Запись в файл
writer = csv.writer(file)
writer.writerow([name, number])

# Закрытие файла
file.close()
